plsql-checks/src/main/java/org/sonar/plsqlopen/checks/ToDateWithoutFormatCheck.java

```diff
@@ -46,9 +49,19 @@ public void init() {
     public void visitNode(AstNode node) {
         MethodMatcher toDate = MethodMatcher.create().name("to_date").addParameter();
         
-        if (toDate.matches(node)) {
+        if (toDate.matches(node) && !argumentIsNull(toDate.getArguments(node) )) {
```

Comment:
CheckUtils.isNullLiteralOrEmptyString could be used here.

You can get the argument with toDate.getArguments(node).get(0).getLastChild() and then call CheckUtils.isNullLiteralOrEmptyString